client/src/planwise/client/projects2/components/settings.cljs

```diff
@@ -183,17 +183,39 @@
                                   :on-change #(dispatch [:projects2/save-key :region-id %])
                                   :model     (:region-id @current-project)
                                   :disabled? read-only}]]))
+
+(defn- set-source-types
```

Comment:
I'd call this `toggle-source-type` instead, and you shouldn't need to check the DOM node's `checked` attribute. You can check the current value of `sources` instead.

Reply:
I'll do it.